src/amsterdam_schema/publish/cli.py

```diff
@@ -148,10 +150,98 @@ def replace_schema_base_url(temp_dir: Path, schema_base_url: str) -> None:
             if file_path.suffix == ".json":
                 with in_place.InPlace(root_path / file_name) as file:
                     for line in file:
-                        line = line.replace(DEFAULT_BASE_URL, schema_base_url)
+                        line = line.replace(DEFAULT_BASE_URL, schema_base_url)  # type: ignore
                         file.write(line)
 
 
+def azure_blob_uploader(
+    files_root: Path,
+    schema_pub_paths: List[List[str]],
+    doc_pub_paths: List[List[str]],
+    container: str,
+    index_file_obj: BytesIO,
+) -> None:
```

Comment:
Missing docstring.

Reply:
Hm. Why oh why is the flake8 plugin not picking this up.